docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -95,3 +101,50 @@ afterSetup(() => {
 </frontmatter>
 ```
 </span>
+
+#### Using Expressive Layout Templates
+
+In the `page.md` file of your layouts, it should come with the following reserved variable:
+
+<box>
+  {{ mainContentBody }}
+</box>
+
+which injects the actual page content in every page. This allows you to build layouts in different ways.
+
+{{ icon_example }}Adding statistics formula to the bottom of every page
+
+<box>
+<i>index.md</i>
+<br><br>
+
+#### Main content of your page
+
+Content of your page
```

Comment:
The rendering of the example looks very odd for both `index.md` and `page.md` because of some inconsistencies in the style:

![image](https://user-images.githubusercontent.com/3168908/72523556-0a2c8e80-389b-11ea-81a7-f03343c356e0.png)

Possible to show the source code of `index.md` and `page.md` instead? `index.html` can still show the rendered result.